jest.db.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    testTimeout: 10000,
+    projects: [
+        {
+            displayName: { name: "db", color: "blue" },
+            testEnvironment: "node",
+            testMatch: ["<rootDir>/itsJustJavascript/db/tests/**/*.test.js"],
+            modulePathIgnorePatterns: ["<rootDir>/wordpress/"],
+        },
+    ],
+}
+
+// special envs for tests
+process.env = Object.assign(process.env, {
+    CATALOG_PATH: "https://owid-catalog.nyc3.digitaloceanspaces.com",
```

Comment:
Which tests is this used in?

Reply:
Leftover from copying this form the other config - removing it